scripts/buildinputs/dockerfile.go

```diff
@@ -31,11 +32,11 @@ func getDockerfileDeps(dockerfile string) string {
 			// random digest value
 			digest:   "sha256:a1c7d58d98df3f9a67eda799200655b923ebc7a41cad1d9bb52723ae1c81ad17",
 			dir:      "/",
-			platform: "linux/amd64",
+			platform: "linux/" + runtime.GOARCH,
```

Comment:
@coderabbitai this will work for arm64, but it won't work for the qemu-user s390x builds, please create github issue for me to improve further

@grdryn thanks!

Reply:
@coderabbitai try posting that issue again